docs/design/applications/apps.md

```diff
@@ -0,0 +1,325 @@
+
+    {
+      "clazz": "org.openkilda.applications.info.InfoMessage",
+      "payload": {
+          "clazz": "org.openkilda.applications.info.apps.FlowApplicationCreated",
+          "flow_id": "string",
+          "tunnel_id": 0,
+          "switch_id": "string",
+          "application": "telescope"
+      },
+      "timestamp": 0,
+      "correlation_id": "string"
+    }
+
+Notification of the flow endpoint to stop watch:
+
+    {
+      "clazz": "org.openkilda.applications.info.InfoMessage",
+      "payload": {
+          "clazz": "org.openkilda.applications.info.apps.FlowApplicationRemoved",
+          "flow_id": "string",
+          "tunnel_id": 0,
+          "switch_id": "string",
+          "application": "telescope"
+      },
+      "timestamp": 0,
+      "correlation_id": "string"
+    }
+
+### Stats
+
+Port statistics:
+
+    {
+      "clazz": "org.openkilda.applications.info.InfoMessage",
+      "payload": {
+          "clazz": "org.openkilda.applications.info.stats.PortStatsData",
+          "switch_id": "string",
+          "stats": [
+             {
+                "clazz": "org.openkilda.applications.info.stats.PortStatsEntry",
+                "port_no": 0,
+                "rx_packets": 0,
+                "tx_packets": 0,
+                "rx_bytes": 0,
+                "tx_bytes": 0,
+                "rx_dropped": 0,
+                "tx_dropped": 0,
+                "rx_errors": 0,
+                "tx_errors": 0,
+                "rx_frame_err": 0,
+                "rx_over_err": 0,
+                "rx_crc_err": 0,
+                "collisions": 0
+             }
+          ]
+      },
+      "timestamp": 0,
+      "correlation_id": "string"
+    }
+
+Flow statistics:
+
+    {
+      "clazz": "org.openkilda.applications.info.InfoMessage",
+      "payload": {
+          "clazz": "org.openkilda.applications.info.stats.FlowStatsData",
+          "switch_id": "string",
+          "stats": [
+             {
+                "clazz": "org.openkilda.applications.info.stats.FlowStatsEntry",
+                "table_id": 0,
+                "cookie": 0,
+                "packet_count": 0,
+                "byte_count": 0,
+                "in_port": 0,
+                "out_port": 0,
+                "match" : {
+                    "src_ip": "string",
+                    "src_port": 0,
+                    "dst_ip": "string"
+                    "dst_port": 0,
+                    "proto": "string",
+                    "eth_type": "string"
+                }
+             }
+          ]
+      },
+      "timestamp": 0,
+      "correlation_id": "string"
+    }
+
+Table statistics:
+
+    {
+      "clazz": "org.openkilda.applications.info.InfoMessage",
+      "payload": {
+          "clazz": "org.openkilda.applications.info.stats.SwitchTableStatsData",
+          "switch_id": "string",
+          "table_stats_entries": [
+             {
+                "clazz": "org.openkilda.applications.info.stats.TableStatsEntry",
```

Comment:
nit: SwitchTableStatsEntry?

Reply:
nope, it's a tablestats, as it's done in kilda